src/pages/ProductPage/RadioLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import { useTranslation } from 'react-i18next';
+
+import { Box, Typography } from '@mui/material';
+
+function RadioLabel() {
+  const { t } = useTranslation();
+
+  return (
+    <Box display="flex" flexDirection="column" gap="5px">
+      <Typography variant="button">{t('product.oneTimeRental')}</Typography>
+      <Typography variant="h4" sx={{ letterSpacing: '-1px' }}>
+        {t('product.reserve')}
+      </Typography>
+      <Typography variant="body1" sx={{ fontWeight: 500, lineHeight: '21px' }}>
+        {t('product.mockPriceRadio')}
+      </Typography>
+    </Box>
+  );
+}
+
+export default RadioLabel;
```

Comment:
this componet sounds as common, please double check that it on the right place

Reply:
Thank you for comment. For now this component is not used at other places, I would prefer to leave it here for now, but once we will use in other components - I will refactor and place it in components.